src/output-json-email-common.c

```diff
@@ -131,15 +131,10 @@ static void EveEmailLogJSONMd5(OutputJsonEmailCtx *email_ctx, JsonBuilder *js, S
         }
         field = MimeDecFindField(entity, "subject");
         if (field != NULL) {
-            unsigned char md5[SC_MD5_LEN];
-            char smd5[256];
+            char smd5[SC_MD5_HEX_LEN + 1];
             char *value = BytesToString((uint8_t *)field->value , field->value_len);
```

Comment:
this is not in your change, but it caught my eye: why would we even do `BytesToString` here if we then don't use the value as a string but as an array of chars? Seems we could just pass `field->value` and `field->value_len` directly to `SCMd5HashBufferToHex`

Reply:
I have an easy patch to improve this, but do we have a test that covers it?

Reply:
I see SV pr/393 adds one.